lib/runner/TestRunner.js

```diff
@@ -57,6 +57,10 @@ module.exports = class TestRunner {
             throw Error("Locale must be defined either in the testing.json or the test file itself under the config element");
         }
 
+        if (testSuite.filterObject() && testSuite.filterObject().onTestSuiteStart) {
```

Comment:
This should be moved down a few lines - then the locale does not need to be passed but instead can just be retrieved from the testSuite.